### PR TITLE
feat: add getVisibleElements

### LINKED ISSUE #17: Need a method to retrieve current visible elements
## Description
A method is required to return the currently visible elements after calling check().

### PR DESCRIPTION
**Pull request by WoodNeck**
## Issue
#17 

## Details
This resolves #17

### CODE REVIEW

File: src/Visible.js [hunk 1]
```diff
@@ -140,6 +140,15 @@ class Visible extends Component {
 		return this;
 	}
 
+	/**
+	 * Get current visible elements.
+	 * @ko 현재 보이는 엘리먼트들을 반환한다.
+	 * @return {HTMLElement[]} An array of visible elements.<ko>현재 보이는 엘리먼트들의 배열</ko>
+	 */
+	getVisibleElements() {
+		return [].slice.call(this._targets).filter(el => !!el.__VISIBLE__);
```
Comment (mixed) [inline src/Visible.js:149]: Since `visible` supports ie8, it seems that filter should not be used.

**Review by mixed — COMMENTED** [review]
👀

File: src/Visible.js (hunk 1 above)
Comment (WoodNeck) [reply src/Visible.js:149]: Thanks for checking, I'll fix that 👍

**Review by daybrush — APPROVED** [review]
LGTM

**Review by mixed — APPROVED** [review]

**Pull request merged**